iam/sc-enduser-iam.yaml

```diff
@@ -27,18 +27,6 @@ Resources:
               AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
             Action:
               - 'sts:AssumeRole'
-  CFNDenyGetTemplateSummaryPolicy:
-    Type: 'AWS::IAM::ManagedPolicy'
-    Properties:
-      PolicyDocument:
-        Version: '2012-10-17'
-        Statement:
-          -
-            Effect: Deny
-            Action:
-              # Deny this action to hide secrets
-              - cloudformation:GetTemplateSummary
-            Resource: "*"
```

Comment:
Why do you want to remove this entirely? It is not being applied to the group or role above. I anticipate we will want to re-apply this when we track down the problem.

Reply:
this policy seems to apply to all resources?  I know it is deny me and I have nearly all permissions. https://pasteboard.co/ISfBKe3.png

Reply:
Ok let's give it a try. We will have to re-figure out how to hide our SSM secrets though